packages/quantic/force-app/main/default/lwc/quanticCaseClassification/quanticCaseClassification.js

```diff
@@ -338,7 +338,26 @@ export default class QuanticCaseClassification extends LightningElement {
    * @returns {void}
    */
   setFieldValue(value) {
-    this.field.update(value);
+    this.field?.update(value);
```

Comment:
The component can't work without its controller. I understand we might not want the component to break the whole page, but the error must be visible somewhere.